core/modules/parser/QSMySqlListener.cc

```diff
@@ -1240,6 +1336,15 @@ class QservFunctionSpecAdapter :
         }
     }
 
+    void checkContext() const override {
+        // required:
+        assertExecutionCondition(__FUNCTION__, _ctx->QSERV_AREASPEC_BOX() != nullptr ||
+            _ctx->QSERV_AREASPEC_CIRCLE() != nullptr ||
+            _ctx->QSERV_AREASPEC_ELLIPSE() != nullptr ||
+            _ctx->QSERV_AREASPEC_POLY() != nullptr ||
+            _ctx->QSERV_AREASPEC_HULL() != nullptr, "Context check failure.", _ctx);
+    }
```

Comment:
The code formatting could be made a bit better by:
```
assertExecutionCondition(
    __FUNCTION__,
    _ctx->QSERV_AREASPEC_BOX()     != nullptr ||
    _ctx->QSERV_AREASPEC_CIRCLE()  != nullptr ||
    _ctx->QSERV_AREASPEC_ELLIPSE() != nullptr ||
    _ctx->QSERV_AREASPEC_POLY()    != nullptr ||
    _ctx->QSERV_AREASPEC_HULL()    != nullptr,
    "Context check failure.",
    _ctx
);

```

Reply:
I'm not very much a fan of vertical formatting like that.